app/views/users/index.html.erb

```diff
@@ -1 +1,12 @@
 <h1>Here is a list of all user</h1>
+<div class='container'>
+  <% @users.each do |user| %>
+    <% link = "users/#{user.id}" %>
+    <a href=<%= link %>>
+      <div class="user-class">
+        <span class='title'> <%= user.name %></title> <span class='number-of-posts'>Number of posts: <%= user.post_counter %></span>
```

Comment:
- [ ] You have a typo here making an error, It's supposed to be `posts_counter` instead of `post_counter`, So please fix that.

![image](https://user-images.githubusercontent.com/68796232/223888126-575f0bc4-bb86-4431-b9c8-dea9fbd999df.png)